hedera-mirror-test/k6/src/web3/test/contractCallBalance.js

```diff
@@ -0,0 +1,46 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ *
+ * Copyright (C) 2019 - 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import http from "k6/http";
+
+import {TestScenarioBuilder} from '../../lib/common.js';
+import {isNonErrorResponse} from "./common.js";
+
+const url = __ENV.BASE_URL;
+const contract = __ENV.DEFAULT_CONTRACT_ADDRESS
+
+const payload = JSON.stringify({
+  "to":String(contract),
+  "data":"0x6896fabf"
+});
```

Comment:
```suggestion
const payload = JSON.stringify({
  to: `${contract}`,
  data: '0x6896fabf',
});
```
can run `./gradlew test:spotlessJavascriptApply` to fix most format issues